sandpack-react/src/types.ts

```diff
@@ -96,6 +96,7 @@ export interface SandpackOptions {
   editorHeight?: React.CSSProperties["height"];
   classes?: Record<string, string>;
 
+  rtl?: boolean;
```

Comment:
Could you add it to the documentation as well?

Reply:
error with run `dev:landing`

<img width="863" alt="image" src="https://user-images.githubusercontent.com/12525415/221842888-fab52cf0-c340-4e13-bbd9-bd3009ebd63b.png">

Reply:
Error with run `dev:docs`

<img width="730" alt="image" src="https://user-images.githubusercontent.com/12525415/221847486-4a522b83-2921-45fa-9a33-414b2ea19adf.png">

Reply:
Fixing it here https://github.com/codesandbox/sandpack/pull/776

Reply:
docs added